content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+... short discussion about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app"), 
+... link to kubeflow-overview for more details
+... (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+
+... short discussion about how Kubeflow is versioned, 
+... that is, components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+
+<a id="roll-your-own"></a>
+## Path 1: roll your own
+
+... disclaimer that any environment-specific issues are not supported by the Kubeflow community (use a distribution if you want support)
+
+... small discussion of the purpose/function of the kubeflow/manifests repo, 
+... that is, to mirror upstream manifests, and act as a central repository containing the "authoritative" manifests for each kubeflow release, to be used by distributions
+
+... link to kubeflow/manifests readme containing the "roll your own" instructions
+
+<a id="use-a-distribution"></a>
+## Path 2: use a distribution
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Distribution</th>
+        <th>Maintainers</th>
+        <th>Deployment Target</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="/docs/distributions/aws/">Kubeflow on AWS</a></td>
+        <td><a href="https://aws.amazon.com/">AWS</a></td>
```

Comment:
I would suggest removing links that drive users outside of docs to things like the main vendor websites to keep the UX clean.

Reply:
I agree that linking to the main vendor website may not be helpful. I think that it could be helpful to keep the link to the Kubernetes environment could be helpful. Consider linking to the docs instead of the product pages.

Reply:
@joeliedtke I agree, and have removed the links to under the "Maintainers" section.

@RFMVasconcelos Regarding linking to external websites, as I discussed in https://github.com/kubeflow/website/issues/2590#issuecomment-820888767, we should aim to not be hosting any distribution specific docs on `kubeflow.org`.

Users should go to the "Installing Kubeflow" page for a list of current distributions, and get links to them.